manager/notifier.go

```diff
@@ -62,11 +67,13 @@ var (
 	hipchatUrl             = flag.String("notification.hipchat.url", "https://api.hipchat.com/v2", "HipChat API V2 URL.")
 )
 
+type Op int
```

Comment:
`Op` seems a bit ambiguous in a package that does a lot of things. Maybe `notificationOp`?